src/platform/platform.dom.js

```diff
@@ -242,10 +242,7 @@ function createProxyAndListen(chart, type, listener) {
     if (chart.ctx !== null) {
       listener(fromNativeEvent(event, chart));
     }
-  }, chart, (args) => {
-    const event = args[0];
-    return [event, event.offsetX, event.offsetY];
-  });
+  }, chart);
```

Comment:
Initially I had provided some very complex types for `throttled` so that we could verify the arguments to the callback function. However, this identified that we don't actually use the 2nd and 3rd parameters here so I removed the 3rd argument. That change allowed removal of the `updateFn` from the helper which is the breaking change